### DISCUSSION TITLE
liso build fails

### DISCUSSION THREAD

**Discussion by sebaxakerhtc** [ipxe/ipxe#364] [Q&A] [answered]
```
make bin/ipxe.liso EMBED=Legacy.ipxe -C ipxe/src && mv ipxe/src/bin/ipxe.liso /var/www/html/bin/:
make: Entering directory '/ipxe/src'
 [DEPS] image/embedded.c
make: *** No rule to make target 'bin/ipxe.liso'.  Stop.
make: Leaving directory '/ipxe/src'
```

**Comment by NiKiZe** [comment, marked as answer]
What is liso? Searching, oh.
Where did you find any documentation about this? you should probably just build the .iso not .liso

see https://github.com/ipxe/ipxe/commit/a8f037a275886ed533bb0c005bc5464757f76ca7
After that the target was probably not included when the makefiles was refactored.

**Comment by sebaxakerhtc** [comment]
I wanted this because of my laptop, which work just from liso or sdsk :)
[My laptop](https://www.cryptomuseum.com/pc/drs/index.htm)
